cmd/producer/main.go

```diff
@@ -1,17 +1,26 @@
 package main
 
 import (
+	"context"
+	"os"
+	"time"
+
+	kafka "github.com/ONSdigital/dp-kafka"
 	"github.com/ONSdigital/dp-observation-importer/event"
 	"github.com/ONSdigital/dp-observation-importer/schema"
-	"github.com/ONSdigital/go-ns/kafka"
-	"time"
 )
 
 func main() {
+	ctx := context.Background()
 	var brokers []string
 	brokers = append(brokers, "localhost:9092")
 
-	producer, _ := kafka.NewProducer(brokers, "observation-extracted", int(2000000))
+	pChannels := kafka.CreateProducerChannels()
+
+	producer, err := kafka.NewProducer(ctx, brokers, "observation-extracted", int(2000000), pChannels)
+	if err != nil {
+		os.Exit(1)
```

Comment:
log something here?

Reply:
Im 99% sure that this producer never gets run and think it will need to be deleted. I think historically it was used to test the go-ns kafka library